src/builder/QuarkBuilder.sol

```diff
@@ -51,6 +51,170 @@ contract QuarkBuilder {
 
     /* ===== Main Implementation ===== */
 
+    struct BorrowIntent {
+        uint256 amount;
+        string assetSymbol;
+        uint256 blockTimestamp;
+        address borrower;
+        uint256 chainId;
+        uint256[] collateralAmounts;
+        string[] collateralTokenSymbols;
```

Comment:
We've decided to name these `assetSymbols`, so `collateralAssetSymbols`. There's still some existing vars that use `tokenSymbol`, but the goal is to be consistent and use `assetSymbol` going forward

Reply:
updated all `collateralToken_` to `collateralAsset_`

Reply:
Oh in that case, you'll want to update the action contexts in the backend. We only changed `tokenSymbols` => `assetSymbols` and didn't touch any other references of `token`. 

I think we should just stick with `collateralToken` (not talking about symbol here) since we currently call the action  token (e.g. for borrow, supply, transfer) `token`.

Reply:
restored